src/App.js

```diff
@@ -1,27 +1,31 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Test
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator
+        name1="/"
```

Comment:
- [ ] Please replace the slash with the divide symbol `÷` instead to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://github.com/CesarHerr/math_magicians/assets/68796232/b349cd7f-1be3-4a03-9ef1-780f4cf8dc0f)  | ![image](https://github.com/CesarHerr/math_magicians/assets/68796232/6f4e530a-6216-4bd5-bd7f-e9c6c61fda39)  |

Reply:
oh, your right!!, I'll do it right now, thanks for your review! 👍